Assets/Scripts/GameController.cs

```diff
@@ -1,10 +1,185 @@
 ﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+
 
 public class GameController : MonoBehaviour {
 
+	private Token[][] _tokens;
+	private int _gridHeight;
+	private int _gridLength;
```

Comment:
so I wouldn't worry too much about GameController yet - that's mainly the file still in active development. Just wanted to get you guys able to play around